src/Controller/EasyAdminController.php

```diff
@@ -11,118 +12,10 @@
 
 class EasyAdminController extends BaseEasyAdminControler
 {
-    public static function getSubscribedServices(): array
+    use AdminExtensionControllerTrait;
+	
+	public static function getSubscribedServices(): array
```

Comment:
indentation

Reply:
I don't know why it show up like this on github, looking at the file directly the indentation looks fine : https://github.com/alterphp/EasyAdminExtensionBundle/blob/d80dcb9eb9abec49ce0431f3a3f0846cf921481f/src/Controller/EasyAdminController.php

Reply:
I think there are tabulations in the commit instead of 4-spaces. Look at raw file https://raw.githubusercontent.com/alterphp/EasyAdminExtensionBundle/d80dcb9eb9abec49ce0431f3a3f0846cf921481f/src/Controller/EasyAdminController.php

Reply:
Oh right I did not see that, I just fixed it